web/src/pages/CheckoutPage/CheckoutPage.js

```diff
@@ -0,0 +1,30 @@
+import { useQuery } from '@redwoodjs/web'
+
+const GET_STRIPE_CUSTOMER_NAME = gql`
+  query getStripeCustomer($sessionURL: String!) {
+    getStripeCustomer(sessionURL: $sessionURL)
+  }
+`
+
+const CheckoutPage = () => {
+  const sessionURL = getURLParameter()
+  const { data } = useQuery(GET_STRIPE_CUSTOMER_NAME, {
+    variables: { sessionURL },
+  })
+
+  return (
+    <div className="flex justify-center">
+      <span className="text-2xl font-bold text-green-800 p-10 rounded-xl bg-green-200 m-0">
+        {`Thanks for signing up, ${data?.getStripeCustomer}!`}
+      </span>
+    </div>
+  )
+}
+
+const getURLParameter = () => {
+  const pageURL = window.location.search.substring(1)
+  const sessionName = pageURL.split('=')
+  return sessionName[1]
+}
```

Comment:
We can leave this for now but it's quite naïve. What if the query param is `fake=1&sessionId=2`, will we get the session ID?

Maybe we could build a generic `getURLParameter` function in a util file. It could take two parameters, a URL and the param you want to extract from the URL (in this case `sessionId`).